docs/UserGuide.md

```diff
@@ -30,11 +30,146 @@ It helps to keep track of patient data, patient appointments and patient bills f
 
 ## 1.1 Add
 
-### 1.1.1 Adding a patient
+### 1.1.1 Adding a patient `addpatient`, `ap`
 
-### 1.1.2 Adding an appointment of a patient
+Adds a patient to the address book with input information including name, phone number,
+email address, home address, remarks and tags. 
 
-### 1.1.3 Adding a bill of an appointment
+* Name must be different from existing patient. 
+
+* Remark and tags are optional. 
+
+* A patient can be added with multiple tags.
+
+#### Command word 
+
+`addpatient`, `ap`
+
+#### Format
+
+`Command word + <prefix><input> ...`
+
+#### Parameter List
+
+|     | Prefix  | Meaning       | Input Constraint                                                                                       |
+|-----|---------|---------------|--------------------------------------------------------------------------------------------------------|
+| `*` | `n/`    | Name          | 1. Non-empty alphanumeric characters and spaces<br/> 2. Must be different from existing patient's name |
+| `*` | `p/`    | Phone number  | Numbers with at least 3 digits                                                                         |
+| `*` | `e/`    | Email address | `local-part@domain`                                                                                    |
```

Comment:
i dont think local-part is understood by user

Reply:
Same one in MESSAGE_CONSTRAINT